common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -255,6 +255,98 @@ color_lists = {
 			125 = "black"
 			250 = "black_empire_dark_red"
 		}
+		
+		###### Warcraft Culture Specific CoAs ######
+		
+		### Humans ###
+		
+		special_selection = { 
+			trigger = { azerothian_coa_trigger = yes }
+			15  = "green_tirassian"
+			15  = "cyan_silverhand"
+			20  = "purple"
+			20  = "red"
+			150 = "blue"
```

Comment:
So I would rise weights so the max weight is 250. I would say right now they're barely noticeable.

Reply:
Can do. The reason I set them so low was to allow a bit of the base colours to seep through occasionally in order to give some variety. (This was more a concern when each culture had 2 or 3 colours only)